protostar/starknet/types.py

```diff
@@ -1,2 +1,9 @@
+from typing import Union
+
 SelectorType = int
 ClassHashType = int
+
+CairoDataRepresentation = list[int]
```

Comment:
Those types belong to `protostar/starknet_gateway/data_transformer.py`, they are not related to starknet os. `Wei` can stay here

Reply:
They are related to StarkNet. For example, the testing cheatcode `deploy_contract` uses data transformer, but it doesn't use StarkNet client.

Reply:
The data transformer core is in `protostar/starknet/data_transformer.py`.

Reply:
Internal discussion. Renamed `DataTransformer` to `DataTransformerPolicy`.